vapi/appliance/shutdown/shutdown.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0.
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package shutdown
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/vmware/govmomi/vapi/rest"
+)
+
+const (
+	ApplianceShutDownPath = "/appliance/shutdown"
+	action                = "action"
+	Cancel                = "cancel"
```

Comment:
do we need to export these?

Reply:
So should `action` also be exported since you currently use string literals in the code instead of a `const`

Reply:
changed 
`action                = "action" to Action                = "action"`
referring this in the simulator.go as 
`switch r.URL.Query().Get(shutdown.Action) {
}`